maven-failsafe-plugin/pom.xml

```diff
@@ -127,6 +127,19 @@
               <excludeTransitive>true</excludeTransitive>
             </configuration>
           </execution>
+          <execution>
+            <id>shared-logging-generated-sources</id>
```

Comment:
IMHO, a little comment to explain why the source code is rebuilt here could be useful for future maintainers